spec/features/spree/admin/taxjar_settings_spec.rb

```diff
@@ -9,6 +9,7 @@
 
   describe "Taxjar settings tab" do
     let(:api_token) { "token" }
+    let(:order) { create :order }
```

Comment:
I don't think this variable is referenced.

Reply:
It's referenced here!

https://github.com/SuperGoodSoft/solidus_taxjar/blob/8689f6c871f7c52baf86e30f403f7caa78529c33/spec/features/spree/admin/taxjar_settings_spec.rb#L31